src/components/FindMovie/FindMovie.tsx

```diff
@@ -15,43 +73,55 @@ export const FindMovie: React.FC = () => {
               data-cy="titleField"
               type="text"
               id="movie-title"
+              value={query}
+              onChange={handlerOnChange}
               placeholder="Enter a title to search"
               className="input is-dander"
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames('button', 'is-light', {
+                'is-loading': isLoading,
+              })}
+              disabled={query === ''}
             >
               Find a movie
             </button>
           </div>
 
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
-            >
-              Add to the list
-            </button>
-          </div>
+          {movie && (
+            <div className="control">
+              <button
+                data-cy="addButton"
+                type="button"
+                className="button is-primary"
+                onClick={() => handlerOnClick()}
```

Comment:
No need additional callback
```suggestion
                onClick={handlerOnClick}
```